aten/src/ATen/native/native_functions.yaml

```diff
@@ -705,7 +705,7 @@
     SparseCPU: empty_sparse
     SparseCUDA: empty_sparse
 
-- func: _empty_affine_quantized(int[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, float scale=1, int zero_point=0) -> Tensor
+- func: _empty_affine_quantized(int[] size, float scale=1, int zero_point=0, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
in general it's not  BC-compatible to reorder args, but  for now it's fine as it's not released yet

Reply:
I changed order because test for q_scale failed and when I looked at other functions in native_functions.yaml, all the TensorOption arguments are the last argument of the function.

Reply:
switching back since the test passed with original order